.github/workflows/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10', "3.11"]
```

Comment:
```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```
Or double quotes everywhere.

Reply:
We can apply prettier on yml now that there won't be conflict as we merged #536

Reply:
https://github.com/PyCQA/prospector/pull/541